ggshield/core/git_hooks/ci/commit_range.py

```diff
@@ -213,13 +208,17 @@ def github_actions_range(verbose: bool) -> List[str]:  # pragma: no cover
             err=True,
         )
 
-    if push_before_sha and push_before_sha != EMPTY_SHA:
-        commit_list = get_list_commit_SHA(f"{push_before_sha}...")
+    # The PR base sha has to be checked before the push_before_sha
+    # because the first one is only populated in case of PR
+    # whereas push_before_sha can be populated in PR in case of
+    # push force event in a PR
+    if pull_req_base_sha and pull_req_base_sha != EMPTY_SHA:
+        commit_list = get_list_commit_SHA(f"{pull_req_base_sha}..")
         if commit_list:
             return commit_list
 
-    if pull_req_base_sha and pull_req_base_sha != EMPTY_SHA:
-        commit_list = get_list_commit_SHA(f"{pull_req_base_sha}..")
+    if push_before_sha and push_before_sha != EMPTY_SHA:
+        commit_list = get_list_commit_SHA(f"{push_before_sha}...")
```

Comment:
I guess you changed from `..` to `...` on purpose. Can you add a comment explaining why?

Reply:
I did not change `..` to `...`, I modified the order of the if statements, I test `pull_req_bse_sha` before the `push` now, as the `push_before_sha` can be populated in PR envs.

Reply:
Ah right, I missed it :+1:.